_posts/2025-03-13-debezium-3-1-beta1-released.adoc

```diff
@@ -0,0 +1,270 @@
+---
+layout: post
+title:  Debezium 3.1.0.Beta1 Released
+date:   2025-03-13
+tags: [ releases, mongodb, mysql, mariadb, postgres, sqlserver, cassandra, oracle, db2, vitess, outbox, spanner, jdbc, informix, ibmi ]
+author: ccranfor
+---
+
+We are pleased to announce the first beta release of Debezium 3.1, **3.1.0.Beta1**.
+
+This release includes a myriad of features and improvements, including but not limited to our first official release of Debezium Server UI, CloudEvent traceparent support, new features for Debezium's PubSub and RabbitMQ sinks, schema access in the WASM transformation, and much, much more.
+Let's dive in and take a look at all these new features and improvements.
+
++++<!-- more -->+++
+
+* link:#new-features-and-improvements[New Features and Improvements]
+* link:#other-changes[Other Changes]
+
+[id="new-features-and-improvements"]
+== New features and improvements
+
+The upgrade to Debezium **3.1.0.Beta1** introduces several new features and improvements in several components:
+
+=== First release of Debezium Server UI
+
+A year ago we began this incredible journey to create a new, modern user interface for Debezium Server that aimed to ease the deployment of Debezium on Kubernetes.
+We are excited to announce that Debezium 3.1 will be the first official release of this years-long effort.
+
+The new UI provides a modern pipeline-based approach to designing source and sink configurations, transformation chains, and more within seconds.
+You can download the Debezium Server UI artifacts directly from https://quay.io[quay.io] as follows:
```

Comment:
Since we have an official release I'll add the helm chart stuff since the single images are not going to work. 

`helm install debezium-platform --set domain.url=<your domain> --version 3.1.0-beta1 oci://quay.io/debezium-charts/debezium-platform`

and then add reference to https://github.com/debezium/debezium-platform/blob/main/helm/README.md